spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -0,0 +1,254 @@
+require 'rails_helper'
+
+shared_examples 'fetching the expected notes' do
+  it 'responds with expected notes' do
+    expect(response_body.to_json).to eq(expected)
```

Comment:
```suggestion
    expect(response_body).to eq(expected)
```
Personalmente, prefiero que el valor que llegue con expect, no sea modificado. Si tenes problemas con diferencias en el hash esperado y el recibido, avisanos! Pero considero esto buena práctica, para que el que lea el test, pueda entender perfectamente que se espera del flujo.